core/src/test/java/org/elasticsearch/plugins/PluginManagerUnitTests.java

```diff
@@ -123,10 +123,9 @@ public void testTrimmingElasticsearchFromOfficialPluginName() throws IOException
     @Test
     public void testTrimmingElasticsearchFromGithubPluginName() throws IOException {
         String user = randomAsciiOfLength(6);
-        String randomName = randomAsciiOfLength(10);
-        String pluginName = randomFrom("elasticsearch-", "es-") + randomName;
+        String pluginName = randomAsciiOfLength(10);
```

Comment:
Just a question because I don't understand how the current github shortening heuristics work. If we don't shorten those names anymore, what does this change for user plugin providers?